tests/adapter/dbt/tests/adapter/basic/test_validate_connection.py

```diff
@@ -0,0 +1,13 @@
+import dbt.task.debug
+
+
+class BaseValidateConnection:
+    def test_validate_connection(self, dbt_profile_data):
+
+        dbt.task.debug.DebugTask.validate_connection(
+            dbt_profile_data["test"]["outputs"]["default"]
+        )
+
+
+class TestValidateConnection(BaseValidateConnection):
+    pass
```

Comment:
Is this class necessary?

Reply:
Yup, we do it like this so the adapters can inherent the base class and also run the tests